tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -123,9 +157,39 @@ _root_.scala.Int#`+`(I)I. => abstract def +: (x: Int): Int
   [10..13): Int => _root_.scala.Int#
 _root_.scala.Int#`<init>`()V. => primaryctor <init>: (): Int
   [4..7): Int => _root_.scala.Int#
+_root_.scala.collection.GenTraversableLike#headOption()Lscala/Option;. => abstract def headOption: Option[A]
+  [0..6): Option => _root_.scala.Option#
+  [7..8): A => _root_.scala.collection.GenTraversableLike#[A]
+_root_.scala.collection.GenTraversableLike#map(Lscala/Function1;Lscala/collection/generic/CanBuildFrom;)Ljava/lang/Object;. => abstract def map: [B, That] => (f: Function1[A, B])(implicit bf: CanBuildFrom[Repr, B, That]): That
```

Comment:
These symbols are not overridden in the document, so they should not be included here even if `:overrides:all`